tests/src/test/scala/whisk/core/invoker/ActionLogDriverTests.scala

```diff
@@ -77,8 +77,8 @@ class ActionLogDriverTests
             raw"""|{"time":"","stream":"stdout","log":"a"}
                   |{"time":"","stream":"stdout","log":"b"}
                   |{"time":"","stream":"stdout","log":"c"}
-                  |{"time":"","stream":"stdout","log":"$LOG_ACTIVATION_SENTINEL\n"}
-                  |{"time":"","stream":"stderr","log":"$LOG_ACTIVATION_SENTINEL\n"}""".stripMargin('|')
+                  |{"time":"","stream":"stdout","log":"$ActionLogDriver.LOG_ACTIVATION_SENTINEL\n"}
+                  |{"time":"","stream":"stderr","log":"$ActionLogDriver.LOG_ACTIVATION_SENTINEL\n"}""".stripMargin('|')
```

Comment:
How do those still pass? Doesn't this need to be `${ActionLogDriver.LOG_ACTIVATION_SENTINEL}`